agent/bench-scripts/tests/pbench-specjbb2005/test-56.pre

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/bash
+
+_dir="${_testtmp}/specjbb2005"
+mkdir ${_dir} || exit 1
+touch ${_dir}/SPECjbb.props || exit 1
+printf "input.jvm_instances=_replace_\n" >> ${_dir}/SPECjbb.props
+printf "input.measurement_seconds=_replace_\n" >> ${_dir}/SPECjbb.props
+printf "input.starting_number_warehouses=_replace_\n" >> ${_dir}/SPECjbb.props
+printf "input.increment_number_warehouses=_replace_\n" >> ${_dir}/SPECjbb.props
+printf "input.ending_number_warehouses=_replace_\n" >> ${_dir}/SPECjbb.props
```

Comment:
Argh; it took me too much time to realize you're processing a template using a `sed` pattern where the value after the `=` doesn't matter, and this caused some confusion. I don't really have a specific alternative since clearly this `sed` pattern has been accepted for a while. Maybe even just a `# benchmark wrapper replaces everything after =` to clarify what's really happening here?

Reply:
Sure, I'll add a comment.